src/pages/AddProduct/AddProduct.tsx

```diff
@@ -1,12 +1,109 @@
-import React from "react";
+import "./AddProduct.scss"
+import React, { useState } from "react";
+import logo from "./images/books.png";
+type Product = {
+  title: string;
+  isbn: string;
+  pageCount: number;
+};
+export const AddProduct = () => {
+  const [products, setProducts] = useState<Product[]>([]);
+  const [title, setTitle] = useState('');
+  const [isbn, setIsbn] = useState('');
+  const [pageCount, setPageCount] = useState("");
 
-export const addProduct = () => {
+  const addProduct = async () => {
+    try {
+      const res = await fetch("https://fakeapi.extendsclass.com/books", {
+        method: "POST",
+        headers: { "Content-Type": "application/json",
+         },
 
+        body: JSON.stringify({
+          title: title,
```

Comment:
if you have the same name of prop and variable you can do it shorter > title: title > title, the same below isbn etc.

Reply:
How can I write this shorter? I have a title: title so I don't understand the mistake :(

Reply:
body: JSON.stringidy({
title, 
isbn,
...

Reply:
Done